### PR TITLE
[Symfony] CVE-2016-1902

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

**Comment by fabpot** [comment]
The validator does not pass

File: symfony/security-core/CVE-2016-1902.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+title:     "CVE-2016-1902: SecureRandom's fallback not secure when OpenSSL fails "
+link:      http://symfony.com/blog/cve-2016-1902-securerandom-s-fallback-not-secure-when-openssl-fails
+cve:       CVE-2016-1902
+branches:
+    2.4.x:
+        time:     2016-01-14 09:59:32
+        versions: [>=2.4.0,<2.5.0]
```
Comment (GrahamCampbell) [inline symfony/security-core/CVE-2016-1902.yaml]: Missing single quotes in all cases.

Comment (GrahamCampbell) [inline symfony/security-core/CVE-2016-1902.yaml]: eg https://github.com/FriendsOfPHP/security-advisories/blob/master/symfony/routing/2012-08-28.yaml

**Comment by xabbuh** [comment]
syntax fixed

**Pull request merged**